gemini/prompts/prompt_optimizer/get_started_with_vertex_ai_prompt_optimizer_sdk.ipynb

```diff
@@ -0,0 +1,1161 @@
+{
```

Comment:
<p>It's better to use vertexai package if possible, for example:</p><p>import vertexai</p><p>vertexai.init()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
This time I have to run a custom job.